virttest/env_process.py

```diff
@@ -1041,6 +1044,13 @@ def preprocess(test, params, env):
                     env["cpu_driver"] = cpu_driver[0]
                     params["cpu_driver"] = env.get("cpu_driver")
 
+    kvm_module_params = params.get("kvm_module_parameters", "DEFAULT")
+    if not kvm_module_params == "DEFAULT":
+        global _kvm_module_handler
+        _kvm_module_handler = KernelModuleHandler('kvm')
+        kvm_module_force = params.get("kvm_module_force_load", "no") == "yes"
```

Comment:
I can see multiple use-cases here:

1. make sure only my params are set
2. require at least given param

For me the (2) is the best candidate as I want to make sure nested is enabled, but I don't want to reload the module if it already supports this.

The (1) is simpler for cases where I know exactly my params, but it can lead to unexpected module reloads.

Reply:
Also note that in case of (2) we might argue whether it should only override the `params` and keep the other params as they were, or whether it should use defaults.

I don't really care whether to use defaults or the current values for the undefined params, but I'd really prefer to only reload the module when the `params` differ.

Reply:
> I can see multiple use-cases here:
> 
> 1. make sure only my params are set
> 2. require at least given param
> 
> For me the (2) is the best candidate as I want to make sure nested is enabled, but I don't want to reload the module if it already supports this.
> 
> The (1) is simpler for cases where I know exactly my params, but it can lead to unexpected module reloads.

Though (1) looks like a valuable addition I'd like to defer implementation as out-of-scope for this PR.